docs/developer-guide/contributing/pull-requests.md

```diff
@@ -112,7 +112,7 @@ With your code ready to go, this is a good time to double-check your submission
 * Make separate pull requests for unrelated changes. Large pull requests with multiple unrelated changes may be closed without merging.
 * All changes must be accompanied by tests, even if the feature you're working on previously had no tests.
 * All user-facing changes must be accompanied by appropriate documentation.
-* Follow the [Code Conventions](../code-conventions.html).
+* Follow the [Code Conventions](../code-conventions.md).
```

Comment:
These markdown pages are generated into the eslint website in [this repository](https://github.com/eslint/website), so this is actually correct. You can see the page [here](https://eslint.org/docs/developer-guide/contributing/pull-requests). I don't think `.html` is necessary, but it also doesn't hurt, either - we seem to do that elsewhere in the documentation as well.